commons/src/main/java/org/infinispan/commons/util/Features.java

```diff
@@ -0,0 +1,53 @@
+package org.infinispan.commons.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.Properties;
+
+import org.infinispan.commons.logging.Log;
+import org.infinispan.commons.logging.LogFactory;
+
+/**
+ * Features allow conditional enabling/disabling of Infinispan's functionality.
+ * They are represented as named properties of the form 'org.infinispan.feature.*' and can either be set via one or
+ * more 'META-INF/infinispan-features.properties' file on the classpath or by setting a system property,
+ * e.g. -Dorg.infinispan.feature.A=true
+ */
+public class Features {
+   private static final Log log = LogFactory.getLog(Features.class);
```

Comment:
Unused

Reply:
I am now using it by listing loaded features.